docs/setup-react-native.md

```diff
@@ -81,3 +81,25 @@ try {
   console.log(error)
 }
 ```
+
+## Using BBS Signatures
+
+When using AFJ inside the React Native environment, temporarily, a dependency for creating keys, sigining and verifying
+with bbs keys must be swapped. Inside your package.json the following must be added:
+
+```diff
++   "resolutions": {
++     "@mattrglobal/bbs-signatures": "react-native-bbs-signatures@0.1.1",
```

Comment:
Resolution field only work when using yarn. We should add an example for the NPM equivalent: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides (would be good to test if this works).

I think bifold uses npm. We also need to mention you need at least NPM 8 for the `overrides` to work (if using NPM)

Reply:
Confirmed to work:


```
"node_modules/@mattrglobal/bbs-signatures": {
      "resolved": "node_modules/@mattrglobal/jsonld-signatures-bbs/@animo-id/react-native-bbs-signatures@0.1.0",
      "link": true
},

```